charts/prometheus-blackbox-exporter/templates/_helpers.tpl

```diff
@@ -150,10 +150,10 @@ securityContext:
 {{- end }}
 {{- with .Values.extraInitContainers }}
 initContainers:
-  {{- toYaml . | indent 2 }}
+{{ toYaml . | indent 2 }}
 {{- end }}
 containers:
-{{- with .Values.extraContainers }}
+{{ with .Values.extraContainers }}
```

Comment:
```suggestion
{{- with .Values.extraContainers }}
```

The difference between the {{ syntax and the {{- syntax is that the {{- something }} will result in space on the left being removed. Without this any extra space would be included which could result in incorrectly formatted YAML.